### PR TITLE
Create STYLEGUIDE.md

### LINKED ISSUE #19134: Add documentation style guide
The style guide topic should be alongside the documentation contribution topic.

It should start with what the base style guide is (I like Microsoft Style Guide). It will include things like:

- Places where the Grafana style guide differs with MSG (like click vs. tap vs. select).
- Quick reference for word usage, heading guidelines, and so on.

### PR DESCRIPTION
**Pull request by oddlittlebird**
Fixes #19134

### CODE REVIEW

**Review by bharjala — CHANGES_REQUESTED** [review]
Two initial suggestions:
- Add a link to [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html)
- Add an example for active / passive voice, similar to your example for imperative second person.
- - Add an example of sentence case, headings that start with verbs, etc. 

For new tech writers and contributors, clear, upfront examples are helpful.

**Comment by marcusolsson** [comment]
I've been looking at the current images:

Preferred format: PNG
DPI: Most images are 72 DPI with some outliers

Width range: 32px – 3712px (most between 430px – 1290px)
Height range: 30px – 1854px (most between 30px – 910px)

Biggest one seems to be: https://github.com/grafana/website/blob/master/static/static/img/docs/v60/explore_split.png

For now, maybe set maximum size to 3840×2160 (4K), for high-quality screenshots?

As for the minimum size, I'm thinking 256x256. There are a handful of images smaller than that, but I'm debating whether that's too small. Here's an example: https://github.com/grafana/website/blob/master/static/static/img/docs/v60/explore_menu.png

**Comment by oddlittlebird** [comment]
Applied edits.

We still lack information for how users should deal with image callouts.

I think we need to give guidelines, tell them to just take the screenshot and we will add callouts, or tell them Grafana personnel will supply updated images.

File: docs/STYLEGUIDE.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+For all items not covered in this guide, refer to the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/) and the [Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html).
+
+## Grafana-specific style
+
+### General
+
+* Use active voice. Avoid passive voice.
+  - Passive: The zombie was killed.
+  - Active: I killed the zombie.
+* Write in the imperative second person. Examples: You can do a thing. Click the panel. Shoot the zombie.
```
Comment (marcusolsson) [inline docs/STYLEGUIDE.md]: I'd like to use a friendlier example than _shoot the zombie_, e.g. eat the cake, pet the dog.

File: docs/STYLEGUIDE.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* Write in present tense.
+  - Not: The panel will open.
+  - Use: The panel opens.
+    or
+    Grafana opens the panel.
```
Comment (marcusolsson) [inline docs/STYLEGUIDE.md]: Maybe put this on same line:

- Not: The panel will open.
- Use: The panel opens, or, Grafana opens the panel.

... or add another Use item:

- Not: The panel will open.
- Use: The panel opens.
- Use: Grafana opens the panel.

File: docs/STYLEGUIDE.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* Concept and reference topic headings should be nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide
+
+### Images
+
+* Preferred format is .png
+* File extension should be all lowercase.
+* Preferred DPI is 72.
+* Assume all graphics will be exclusively viewed on the web.
+* Maximum image size is 3840px X 2160px.
+* Screenshots should be readible, but not too large.
```
Comment (marcusolsson) [inline docs/STYLEGUIDE.md]: readible -> readable?

File: docs/STYLEGUIDE.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+### Capitalization
+
+* Grafana, Loki, and Prometheus are always capitalized unless part of a code block.
+* `amtool` is never capitalized.
```
Comment (marcusolsson) [inline docs/STYLEGUIDE.md]: This is too specific, and not a Grafana term. I think we should remove it, otherwise we'd have to list a lot of these.

**Review by marcusolsson — CHANGES_REQUESTED** [review]

**Comment by marcusolsson** [comment]
Also, I'm debating whether the style guide should go into https://github.com/grafana/grafana/tree/master/style_guides or whether this should be a proper page under "Contribute" in the documentation.

**Comment by marcusolsson** [comment]
I think best place for now is to put it next to the other style guides in the style_guides directory, and link from the docs guide.

File: docs/STYLEGUIDE.md (hunk 3 above)
Comment (oddlittlebird) [reply docs/STYLEGUIDE.md]: ARGH. And that is why we have editors.

**Review by marcusolsson — CHANGES_REQUESTED** [review]
This is great stuff! 

One minor thing before I think we can merge this. None of the other style guides have the `-style-guide` prefix in their filename. I think we should be consistent here:

Basically rename the file to just `docs.md`.

Almost there!

**Comment by oddlittlebird** [comment]
I renamed the style guides to match their titles with the exception of themes.md, since that one is the subject of a different issue.

**Review by marcusolsson — APPROVED** [review]
Looks great! Thanks for helping out with this! 👍

**Comment by oddlittlebird** [comment]
My pleasure.

**Pull request merged**